boards/dactyl_manuform/readme.md

```diff
@@ -0,0 +1,49 @@
+# Dactyl Manuform
+
+The [Dactyl Manuform](https://github.com/tshort/dactyl-keyboard) is a handwired, split bodied, tented, curved keywell keyboard - adapted from the [Dactyl](https://github.com/adereth/dactyl-keyboard) keyboard with the thumb cluster design of the [ManuForm](https://geekhack.org/index.php?topic=46015.0) keyboard. Information for building a Dactyl Manuform found in first link.
+
+![Imgur](https://i.imgur.com/7y0Vbyd.jpg)
+*Two wired Dactyl Manuform 4x6 variants*
+
+## Variants
+
+Dactyl Manuform's are built in variations that cater for different row and column counts, and thumb clusters.  
+
+As standard: 
+- The finger keywell bottom row has 2 keys, 1 each in ring and middle columns
+    - Exception to this rule is the 7 column variants, that have two additional keys in this row
+- The thumb cluster has 6 keys, arranged in 2 columns by 3 rows 
+
+Variants are denoted as `RowCount`*x*`ColumnCount`
+
+### Row
+| Count | Description |
+| --- | --- |
+| 4 | Three rows, typically for alphabet and some puncuation characters, with 2 key (finger keywell) bottom row |
+| 5 | As *4 row* with number row above |
+| 6 | As *5 row* with function row above |
+
+### Column
+| Count | Description |
+| --- | --- |
+| 5 | A column for each finger with additional column for first finger |  
+| 6 | As *5 column* with additional pinky finger column |
+| 7 | As *6 column* with either an additional first finger column (`5x7`) or additional pinky column (`6x7`) |
+
+## Microcontroller support
+
+Amend `controller` in the following line of desired variant's `kb.py` file to supported microcontroller listed in `kmk/quickpin/pro_micro`:
```

Comment:
If you have to edit it, what's the point of having 6 different `kb.py`s in the first place? Why not a single one for 4x5 with the additional rows/colums commented out?
Edit: I guess the `coord_mapping` would be an issue, althogh most of that could be alleviated by reordering the column pins. New request: reorder the column pins to make the `coord_map` as linear as possible.

Reply:
Sorry @xs5871 , I have no idea what entails.

My understanding is the `coord_mapping` is directly related to pin matrix, thus there is a different coord_mapping per keyboard as their pin matrices are unique.

Would you happen to have a reference to your request?

Reply:
No, sorry, strike that. Didn't consider that these are all split boards. Let's leave it the way it is.